template_docker_image.sh

```diff
@@ -16,10 +16,19 @@ PROJECTTITLE=${PROJECT_TITLE}
 echo "Project title set to $PROJECTTITLE"
 
 if [ ! -z "$DOCKER_PROJECT_TITLE" ]; then
-	PROJECTTITLE="$DOCKER_PROJECT_TITLE"
-	echo "Project title changed to $PROJECTTITLE"
+    PROJECTTITLE="$DOCKER_PROJECT_TITLE"
+    echo "Project title changed to $PROJECTTITLE"
 fi
 
+BUILD_DIR="./$PROJECTTITLE"
```

Comment:
My understanding is that the default `BUILD_DIR` is set to `./$PROJECTTITLE` so that the removal of lines 58-59 is non-breaking.  If this is the case, should the default not be `./$PROJECT_TITLE/app` (since the original code moves into the project, then the app directory)?